projects/frontend/data-pipelines/projects/gui/projects/data-pipelines/README.md

```diff
@@ -0,0 +1,82 @@
+# Taurus Data Pipelines
+
+Data Pipelines help Data Engineers develop, deploy, run, and manage data processing workloads (called "Data Job").
+This library provides UI screens that helps to manage data jobs via Data Pipelines API.
+
+## Implementation
+
+
+### Include the packages and ngrx dependencies
+```shell
+npm i @vdk/{data-pipelines,shared} # Actual library
+npm i @ngrx/{effects,entity,router-store,@ngrx/store} # NgRx (store management)
+npm i @clr/{angular,icons,ui} # Clarity (UI Components like DataGrid)
+
+```
+
+### Include the module and router
+
+1. In `app.module.ts` include ngrx modules and actual Data Pipeline module
+```typescript
+imports: [
+  ...
+  VdkSharedCoreModule.forRoot(), // vdk shared core services
```

Comment:
Not planning ong renaming Taurus* class names, those names are already depended-on prior opening the source code; This is not a problem, there already are quite some packages and classes in the repo named after Taurus, it was the legacy name before open-sourcing.

Reply:
Ok cool!